### PR TITLE
chore: change report logic

### PR DESCRIPTION
**Pull request by JoeWang1127**
In this PR:
- Show the package information report regardless of the analysis result.
- Change report format (verified in unit tests).

### CODE REVIEW

File: java-shared-dependencies/dependency-analyzer/src/test/java/com/google/cloud/model/AnalysisResultTest.java [hunk 1]
```diff
@@ -52,54 +52,104 @@ public void testGenerateReportWithoutRiskSucceeds()
...
         Please copy and paste the package information below to your ticket.
                 
         ## Package information of com.example:artifact:1.2.3
-        Licenses: [Apache-2.0]
-        Vulnerabilities: None.
+        Licenses: [Apache-2.0 (Google-compliant)]
+        Vulnerabilities: [].
         Checked in [com.example:artifact (1.2.3)](https://api.deps.dev/v3/query?versionKey.system=MAVEN&versionKey.name=com.example:artifact&versionKey.version=1.2.3)
                 
         ## Dependencies:
```
Comment (suztomo) [inline java-shared-dependencies/dependency-analyzer/src/test/java/com/google/cloud/model/AnalysisResultTest.java:91]: Remove ":" in headings

File: java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/DependencyAnalyzer.java [hunk 2]
```diff
@@ -109,7 +112,8 @@ public static void main(String[] args) throws IllegalArgumentException {
       System.exit(1);
     }
 
-    ReportResult result = analyzeReport.generateReport();
+    System.out.println(analyzeReport.toString());
```
Comment (suztomo) [inline java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/DependencyAnalyzer.java:115]: Insert `System.out.println("Please copy and paste the package information below to your ticket.")`.

The string representation of analyzeReport should not contain the message.

**Review by suztomo — APPROVED** [review]
Added few comments.

Nice use of string block. It's easy to understand the outcome of the implemented logic.

File: java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/DependencyAnalyzer.java (hunk 2 above)
Comment (JoeWang1127) [reply java-shared-dependencies/dependency-analyzer/src/main/java/com/google/cloud/DependencyAnalyzer.java:116]: Done.

File: java-shared-dependencies/dependency-analyzer/src/test/java/com/google/cloud/model/AnalysisResultTest.java (hunk 1 above)
Comment (JoeWang1127) [reply java-shared-dependencies/dependency-analyzer/src/test/java/com/google/cloud/model/AnalysisResultTest.java]: Done

**Pull request merged**